modules/nextflow/src/main/groovy/nextflow/cli/CmdRun.groovy

```diff
@@ -178,6 +178,9 @@ class CmdRun extends CmdBase implements HubOptions {
     @Parameter(names = '-with-singularity', description = 'Enable process execution in a Singularity container')
     def withSingularity
 
+    @Parameter(names = '-with-apptainer', description = 'Enable process execution in a Apptainer container')
+    def withApptainer
```

Comment:
@pditommaso Working through the CLI code, looks like this CLI option is never used, so it probably doesn't work. Need to review later.

Reply:
Oops, worth opening an issue to track it